demo/pages/menu.py

```diff
@@ -0,0 +1,68 @@
+# menu.py
+
+import dash
+from dash import html
+import dash_material_ui as mui
+from dash import Input, Output, callback
+
+from util import create_component_item
+
+dash.register_page(__name__)
+
+# Define menus with different configurations
+menus = [
+    (
+        "Simple Menu",
+        {
+            "id": "menu-1",
+            "buttonText": "Open Menu",
+            "menuItems": ["Profile", "My account", "Logout"],
+        },
+    ),
+    (
+        "Menu with Disabled Item",
+        {
+            "id": "menu-2",
+            "buttonText": "Options",
+            "menuItems": [
+                {"label": "Option 1"},
+                {"label": "Option 2 (Disabled)", "disabled": True},
+                {"label": "Option 3"},
+            ],
+        },
+    ),
+]
+
+# Create the layout using the menus defined above
+layout = html.Div(
+    mui.Grid(
+        spacing=2,  # Controls the spacing between the menus
+        children=[
+            create_component_item(title, mui.Menu, props) for title, props in menus
```

Comment:
Am I missing something ? Running the demo, this line return an Error:
```
(review-py3.12) [work] dream@Latent:~/repo/dash-mui.git/review/demo$ python app.py

Traceback (most recent call last):
  File "/home/dream/repo/dash-mui.git/review/demo/app.py", line 10, in <module>
    app = Dash(__name__, use_pages=True)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dream/.cache/pypoetry/virtualenvs/review-aK06w5nS-py3.12/lib/python3.12/site-packages/dash/dash.py", line 580, in __init__
    self.init_app()
  File "/home/dream/.cache/pypoetry/virtualenvs/review-aK06w5nS-py3.12/lib/python3.12/site-packages/dash/dash.py", line 659, in init_app
    self.enable_pages()
  File "/home/dream/.cache/pypoetry/virtualenvs/review-aK06w5nS-py3.12/lib/python3.12/site-packages/dash/dash.py", line 2189, in enable_pages
    _import_layouts_from_pages(self.config.pages_folder)
  File "/home/dream/.cache/pypoetry/virtualenvs/review-aK06w5nS-py3.12/lib/python3.12/site-packages/dash/_pages.py", line 444, in _import_layouts_from_pages
    spec.loader.exec_module(page_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/dream/repo/dash-mui.git/review/demo/pages/menu.py", line 41, in <module>
    create_component_item(title, mui.Menu, props) for title, props in menus
                                 ^^^^^^^^
AttributeError: module 'dash_material_ui' has no attribute 'Menu'


```

Reply:
you're missing some steps, see https://github.com/ploomber/dash-mui/issues/3

I was able to run the demo successfully

Reply:
Oh, thank you. Reviewing the PR right now